src/components/InngestStepTools.test.ts

```diff
@@ -173,13 +174,21 @@ describe("sleepUntil", () => {
   test("return Sleep step op code", async () => {
     const future = new Date();
     future.setDate(future.getDate() + 1);
-
     expect(() => sleepUntil(future)).toThrow(StepFlowInterrupt);
     await expect(state.nextOp).resolves.toMatchObject({
       op: StepOpCode.Sleep,
     });
   });
 
+  test("parses ISO strings", async () => {
+    const next = new Date().valueOf() + ms("6d");
+
+    expect(() => sleepUntil(new Date(next))).toThrow(StepFlowInterrupt);
```

Comment:
I think we'd need to do `new Date().toISOString()` to test this there, as currently this is passing a date?